### PR TITLE
Remove indexed payment param from OracleRequest log

### PR DESCRIPTION
**Pull request by thodges-gh**

### CODE REVIEW

**Comment by thodges-gh** [comment]
Needs updates to the Go node.

File: store/models/log_events.go [hunk 1]
```diff
@@ -253,14 +255,32 @@ func (le RunLogEvent) Validate() bool {
 
 // ContractPayment returns the amount attached to a contract to pay the Oracle upon fulfillment.
 func (le RunLogEvent) ContractPayment() (*assets.Link, error) {
-	encodedAmount := le.Log.Topics[RequestLogTopicAmount].Hex()
-	payment, ok := new(assets.Link).SetString(encodedAmount, 0)
+	version, err := le.Log.getTopic(0)
+	if err != nil {
+		return nil, err
```
Comment (j16r) [inline store/models/log_events.go]: Could you add some context here, e.g: `Missing topic 0`

File: store/models/log_events.go [hunk 2]
```diff
@@ -347,29 +374,26 @@ func parseRunLog20190123(log Log) (JSON, error) {
 	return js.Add("functionSelector", OracleFulfillmentFunctionID20190123)
 }
 
-func parseRunLog20190128(log Log) (JSON, error) {
+func parseRunLog20190207(log Log) (JSON, error) {
```
Comment (j16r) [inline store/models/log_events.go]: These versions are a bit hard to distinguish, could we add a semantic description or keyword of some sort? e.g: `20190207_sql_release`

Comment (se3000) [reply store/models/log_events.go]: Added some documentation to each method to indicate how it is different from the previous version.

**Pull request merged**